wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,178 @@
+
+![Visual Studio code's Git Interface](10.JPG "Visual Studio's Git Interface")
+
+After Committing your changes, we ned to sync this on GitHub, to be able to do this, Enter the following in your Command Line Interface :
+
+```prolog
+git push origin <branch>
+```
+and this should upload your changes in GitHub.
+
+## Pull Requests
+
+After making your changes, you must make a "Pull Request" to be able for your changes to be made into the wiki.
+
+Go to your Pull Requests tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+You should see the Pull Requests Page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is your Pull Request Page. Now click on "New Pull Request"
+
+and you should see something like this.
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+Now we want to compare ``ppy:master`` against your changes.
+
+Now Select where did you made your changes in "branch" beside your "head Fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However if you made changes in your Master Branch, The Changes will be shown without changing anything in the head fork options_.
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+**Title** : The Summary of the Changes you've made.
+
+**PR Sub-Message** : the Details of the changes you've made. 
+
+![PR Creation Dialogue](14.jpg "PR Creation Dialogue")
+
+Then Click "Create Pull Request" and wait for a Reviewer to check and possibly, Merge your changes.
+ 
+ Your Changes must follow the [Article Styling Guidelines](wiki/Article_Style_Guide/en.md) to be able to get merged.
+
+## FAQ
+
+### Q: Why is the old osu!wiki not staying?
+
+Due to abysmal performance, the old wiki is getting deprecated
+as stated [here](https://github.com/ppy/osu-wiki/issues/69).
+
+### Q : How would this differ from the old wiki?
+
+- We use GitHub for Collaboration. You can Make mass changes easily and efficiently and you can delay your changes as long as you like.
+- Unlike the old osu!wiki, Changes has to go Pull Requests which are then reviewed by staff or someone with write permissions. This adds a layer of security against spammers and vandals.
+- Linking is done Manually, and must be done using Static Linking.
+- **No Templates**. You must duplicate the Template and paste it accordingly to your file.
+- Deletion of Media is easier and requires no special wiki function. However, this has to go under go the Pull request Protocol and may cause Git Conflicts.
+- Tables are supported bare-minimum by GFM without requiring extensions, unlike Mediawiki. However, this does mean we need to code some tables in HTML in extreme cases.
+- Markdown is sensitive. any wrong Capitalization can break things like Static Links.
+
+### Q: Can we use HTML?
+
+We avoid to use HTML mostly in Wiki but in extreme cases, we allow HTML for Tables and Linking.
+
+
+## Conclusion
+I hope this briefs you on how to contribute to the new osu!wiki. If you have any more questions
```

Comment:
Remove first person. I'm not sure on how to reword this sentence... so if you could, reword it or remove it